qa/acceptance-tests/src/test/java/io/camunda/it/client/UserTaskSearchTest.java

```diff
@@ -779,6 +763,43 @@ void shouldReturnUserTaskVariablesFilteredByIn() {
         .containsExactlyInAnyOrder("processVariable", "subProcessVariable");
   }
 
+  @Test
+  void shouldReturnLargeUserTaskVariableTruncated() {
```

Comment:
```suggestion
  void shouldReturnLargeUserTaskVariableNotTruncated() {
```

Reply:
🙃 The test method names are mixed up, looking at the test code. Not a big deal, I was just shortly confused when looking at the name and the code 😅